process_request/routines.py

```diff
@@ -227,7 +230,7 @@ def __init__(self):
             "GroupingOption_Location": "FirstValue",
             "GroupingOption_ItemInfo5": "FirstValue",
             "UserReview": "No",
-            "SubmitButton": "Submit Request",
+            "SubmitButton": _("Submit Request"),
```

Comment:
I don't think this needs to/should be translated. As far as I know this is just POST data that Aeon needs to process a request but effectively swallows, and I don't think it get saved in a transaction.

Reply:
Yeah, that makes sense to me. I wasn't sure if this would make it to the end user, so I was adding the wrapper to anything I thought could make it to the end user.